src/chunk_append/chunk_append.c

```diff
@@ -82,15 +83,15 @@ ts_chunk_append_path_create(PlannerInfo *root, RelOptInfo *rel, Hypertable *ht,
 			break;
 		case T_MergeAppendPath:
 			children = castNode(MergeAppendPath, subpath)->subpaths;
-			path->cpath.path.pathkeys = subpath->pathkeys;
+			if (ordered)
+				path->cpath.path.pathkeys = subpath->pathkeys;
 			break;
 		default:
 			elog(ERROR, "invalid child of chunk append: %u", nodeTag(subpath));
 			break;
 	}
 
-	Assert(ht->space->num_dimensions <= 2);
-	if (ht->space->num_dimensions == 1)
+	if (path->cpath.path.pathkeys == NULL || ht->space->num_dimensions == 1)
```

Comment:
instead of `path->cpath.path.pathkeys == NULL` is `!ordered` more clear? Is that the right logic?